### DISCUSSION TITLE
Calorie slider bar for common foods.

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#47] [Q&A]
In the app for some common foods there is a slider bar that allows to adjust calories for given serving. Is it implemented in the dashboard? 

Example look up Blueberry muffin in the app and in the dashboard.

![image](https://github.com/user-attachments/assets/c3a67602-95f9-4d56-b125-fdb9242a44cf)

**Comment by jlai** [comment]
Unfortunately, I don't think this is possible with the API. The API doesn't give us the range, nor the ability to modify the calories for an existing food. It also doesn't give us the info to calculate the calories for a given quantity (e.g. 5oz), which is done on the server side for the 3rd party API.

**Reply by gramapp** [reply]
Thanks for looking in to this.
